apps/admin-gui-e2e/src/e2e/facilities/facility-admin.cy.js

```diff
@@ -13,6 +13,7 @@ describe('Facility management with role Facility admin', () => {
 
   before(() => {
     cy.login('FACILITY_MANAGER', 'facilityManager');
+    window.localStorage.setItem('settings', 'true');
```

Comment:
The storage items could be set before all the tests to save code, right? I believe support/e2e.ts is the place for this